docs/user/whats-new.asciidoc

```diff
@@ -8,9 +8,101 @@ check the <<release-notes, release notes>>.
 Previous versions: {kibana-ref-all}/8.6/whats-new.html[8.6] | {kibana-ref-all}/8.5/whats-new.html[8.5] | {kibana-ref-all}/8.4/whats-new.html[8.4] | {kibana-ref-all}/8.3/whats-new.html[8.3] | {kibana-ref-all}/8.2/whats-new.html[8.2] 
 | {kibana-ref-all}/8.1/whats-new.html[8.1] | {kibana-ref-all}/8.0/whats-new.html[8.0]
 
-//NOTE: The notable-highlights tagged regions are re-used in the
-//Installation and Upgrade Guide
+[discrete]
+=== Alerting
 
-// tag::notable-highlights[]
+[discrete]
+==== Webhook connector improvement
+The Webhook connector now supports client certification. You can now use the webhook connector for secured integration with third parties, beyond basic authentication (using usernames and passwords or tokens). 
 
-// end::notable-highlights[]
+[role="screenshot"]
+image::images/webhook-connector.png[A screenshot displaying the new SSL authentication option in the UI, width=70%]
+
+[discrete]
+==== Conditional actions
+When creating an observability alerting rule, you define conditions that must be met to trigger an. Now you can also add conditions that affect the actions associated with an alert. For example, you might choose to send notifications only if the alert is received between defined hours.
```

Comment:
> When creating an observability alerting rule...

Since this affects only Observability, IMO we should move this to https://www.elastic.co/guide/en/observability/8.10/whats-new.html instead. I'll open a PR in that repo.

Reply:
++ thank you!

Reply:
Ok thank you, I'll remove it from this page.